roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -25,9 +25,16 @@
 
 - debug: var=ports
 
+- name: check selinux status
```

Comment:
sorry, may be I wasn't clear. I just meant that we don't need to make the selinux specific cleanup ( i.e. `deny openvswitch_t type in selinux` task) conditional. We can always run it, even if it fails the error is ignored. This reduces a task and simplifies the play.